.github/workflows/sync-api.yml

```diff
@@ -0,0 +1,29 @@
+name: Sync Api
+on:
+  push:
+    paths:
+      - 'staging/src/github.com/clusterpedia-io/api/*'
+jobs:
+  sync-api:
+    name: Sync Api
+    runs-on: ubuntu-latest
+    env:
+      WORKSPACE: ${{ github.workspace }}/src/github.com/clusterpedia-io/clusterpedia
+      GH_TOKEN: ${{ secrets.CLUSTERPEDIA_BOT_TOKEN }}
+      MESSAGE: ${{ github.event.head_commit.message }}
+      REFTYPE: ${{ github.ref_type }}
+      TAGNAME: ${{ github.ref_name }}
+      REF: ${{ github.ref }}
```

Comment:
The environment variables related to sync-api are recommended to be placed together with sync-api

Reply:
the environment variables can not get by  sync-api.sh, if placed together with run sync-api.sh